bundles/org.openhab.binding.yamahamusiccast/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -38,8 +38,23 @@
 				<options>
 					<option value="none">None</option>
 					<option value="net_radio">Net Radio</option>
+					<option value="tuner">Radio</option>
```

Comment:
File _src/main/resources/OH-INF/i18n/yamahamusiccast.properties_ needs to be regenerated after adding new strings. Run `mvn i18n:generate-default-translations` in the bundle folder.

Reply:
Done!

Reply:
@jlaur I encountered one big problem that took me quite a while to solve.
See my newest comment in this PR.